R/iso_grid.R

```diff
@@ -1,11 +1,19 @@
 #' Create hexagonal grid
+#' 
+#' Create a hexagonal grid from an sf object
 #'
 #' @param x An sf object the convex hull of which will determine the hex grid
-#' @param ... Extra arguments passed to the grid generator (TBC)
+#' @param resolution The resolution of the hex grid
+#' @return An sf object
```

Comment:
This returns an `sfc` class object not an `sf` object

Reply:
Ah yes. Fix incoming.